amplify/backend/api/YourClearWayAPI/schema.graphql

```diff
@@ -89,6 +82,23 @@ enum Tag {
     WATER
 }
 
+enum Difficulty {
+    BEGINNER
+    INTERMEDIATE
+    ADVANCED
+}
```

Comment:
nit: not sure if this is something we need to clarify with design/YCW PM or if it's pretty minor but on one place in the design, `BEGINNER` is labeled as `STARTER` while in other cases it's `BEGINNER`, and on the mobile app measurement sheet `ADVANCED` is labeled as `EXPERT`

Reply:
I will ask the designer about and have these fields be consistent.